src/zenml/zen_stores/schemas/component_schemas.py

```diff
@@ -251,7 +251,10 @@ def to_model(
         body = ComponentResponseBody(
             user_id=self.user_id,
             type=StackComponentType(self.type),
-            flavor_name=self.flavor,
+            flavor_name=(
```

Comment:
This value on the response is used in many places in code, and I'm sure some of them require this to be the actual name of the flavor in the registry, not a display name.
This most likely breaks lots of our internal code, but even if it doesn't, I would suggest reverting this change.

This is not called `flavor_display_name`, and the only thing you're doing by changing it is potentially breaking existing code for no reason.

Reply:
@Cahllagerfeld then i will take this out.. please note!